files/default/sql/ddl/3.7.0__initial_tables.sql

```diff
@@ -1244,6 +1244,7 @@ CREATE TABLE `serving` (
                            `revision` varchar(8) DEFAULT NULL,
                            `predictor_resources` varchar(1000) COLLATE latin1_general_cs DEFAULT NULL,
                            `transformer_resources` varchar(1000) COLLATE latin1_general_cs DEFAULT NULL,
+                           `protocol` TINYINT(1) NOT NULL DEFAULT '0',
```

Comment:
```suggestion
                           `api_protocol` TINYINT(1) NOT NULL DEFAULT '0',
```

Reply:
Let's call it API protocol to be more specific. 
There are multiple API protocols: REST, gRPC, ...
REST and gRPC are designed on top of HTTP.